### PR TITLE
[AOTI] Add pybind for tensor_converter util functions

### PR DESCRIPTION
**Pull request by desertfire**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #121745
* __->__ #121744
* #121743
* #121523

Differential Revision: [D54818716](https://our.internmc.facebook.com/intern/diff/D54818716)

### CODE REVIEW

**Comment by desertfire** [comment]
@desertfire has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D54818716).

**Comment by desertfire** [comment]
@desertfire has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D54818716).

File: torch/_C/_aoti.pyi [hunk 1]
```diff
@@ -1,3 +1,17 @@
...
+
+from torch import Tensor
+
 # Defined in torch/csrc/inductor/aoti_runner/pybind.cpp
+
+# Tensor to AtenTensorHandle
+def unsafe_alloc_void_ptr_from_tensors(tensors: List[Tensor]) -> List[c_void_p]: ...
+
+# AtenTensorHandle to Tensor
+def alloc_tensors_by_stealing_from_void_ptr(
```
Comment (chenyang78) [inline torch/_C/_aoti.pyi:12]: Curious why we are adding py bindings for these two functions? Thanks.

Comment (chenyang78) [reply torch/_C/_aoti.pyi:12]: Never mind. I saw its usage from the next PR in the stack. Thanks.

**Review by chenyang78 — APPROVED** [review]

**Pull request closed without merging**